src/plugins/home/kibana.json

```diff
@@ -7,7 +7,7 @@
   "version": "kibana",
   "server": true,
   "ui": true,
-  "requiredPlugins": ["data", "share", "urlForwarding"],
+  "requiredPlugins": ["data", "share", "urlForwarding", "customIntegrations"],
   "optionalPlugins": ["usageCollection", "telemetry"],
```

Comment:
(Even if it doesn't change much now that plugins are no longer be able to be disabled by default) Do we need this to be a required dependency? Seems like a harsh constraint to something as generic as the `home` plugin.